src/include/miopen/solver/implicitgemm_util.hpp

```diff
@@ -213,7 +213,7 @@ static inline bool IsXdlopsSupport(const ExecutionContext& ctx)
     // 2) llvm intrin may has incorrect results
     const bool is_xdlops_supported = StartsWith(ctx.GetStream().GetDeviceName(), "gfx908") ||
                                      StartsWith(ctx.GetStream().GetDeviceName(), "gfx90a") ||
-                                     StartsWith(ctx.GetStream().GetDeviceName(), "gfx94");
+                                     StartsWith(ctx.GetStream().GetDeviceName(), "gfx942");
```

Comment:
```suggestion
                                     ctx.GetStream().GetDeviceName() == "gfx942";
```

For other gpus here this can be changed as well